cmake/configs/default.cmake

```diff
@@ -387,7 +387,7 @@ hunter_default_version(thrift VERSION 0.11.0-p0)
 hunter_default_version(tinydir VERSION 1.2-p0)
 hunter_default_version(tinyxml2 VERSION 6.2.0-p1)
 hunter_default_version(toluapp VERSION 1.0.93-p1)
-hunter_default_version(tomcrypt VERSION 1.17-p3)
+hunter_default_version(tomcrypt VERSION 1.18.2-p0)
```

Comment:
We got conflict here after merge of https://github.com/ruslo/hunter/pull/1574, please resolve.